### ISSUE TITLE
Create project group for ELMS, Open Ulmus, and related projects 

### ISSUE THREAD

**Issue by gusaus** [DrupalOpenLearning/drupaldojo#50]
Once the structure worked out https://github.com/DrupalOpenLearning/drupaldojo/issues/9 we can create a project group to showcase ongoing Dojo sessions, events, sprints, etc. involving ELMS:LN, Open Ulmus and related projects. The group could be managed by @mmilutinovic1313 or other ELMS:LN contributors.

I'm assuming we'd want to have an umbrella group for ELMS Learning Network, rather than having a group for each individual project?

**Comment by mmilutinovic013** [comment]
As per the conversation on Slack I believe this umbrella group is desired.

**Comment by gusaus** [comment]
@mmilutinovic1313 I'm aware there are several related conversations, but this has to do with the group we should set up on http://drupaldojo.com/, rather than the 'umbrella service provider' https://github.com/elmsln/openulmus.org/issues/12. Possibly we're already on the same page on that.

Regardless...I'm assuming 'ELMS Learning Network' would be the umbrella group for all related projects?

**Comment by mmilutinovic013** [comment]
Yes I believe that makes the most sense as they would be the technology / community backing the creation of openulmus sites

**Comment by mmilutinovic013** [comment]
Now that we have @cleverington assuming a PM role -- I have updated associated wikis. We currently need to work out a strategy (a "so-what") for Drupal Dojo. I believe one may already exist, but let's define that here @gusaus, @cleverington so that we can add it to the description and wiki. This will also help guide social media and advertising for the dojo.

**Comment by gusaus** [comment]
@mmilutinovic1313 Let's keep this issue focused on setting up the ELMS group on the Dojo site. Possibly we can walk through how to set up groups https://github.com/DrupalOpenLearning/drupaldojo/issues/23 in #drupal-dojo and/or do a working session over twitch at some point.

We're also planning to set up a Dojo group where we can have a public space to strategize and get additional folks involved with running the program https://github.com/DrupalOpenLearning/drupaldojo/issues/58

Let's continue to sort out Dojo roles here https://github.com/DrupalOpenLearning/drupaldojo/issues/5
